independent-publisher-connectors/Safaricom Mpesa Daraja/apiDefinition.swagger.json

```diff
@@ -0,0 +1,620 @@
+                  "type": "string",
+                  "description": "ResponseDescription"
+                },
+                "CustomerMessage": {
+                  "type": "string",
+                  "description": "CustomerMessage"
+                }
+              }
+            }
+          }
+        },
+        "summary": "lipa na mpesa",
+        "description": "lipa na mpesa",
+        "operationId": "LipaNaMpesa",
+        "parameters": [
+          {
+            "name": "content-Type",
+            "in": "header",
+            "required": false,
+            "type": "string",
+	    "x-ms-summary": "lipa na mpesa stk puish",
+	    "description": "lipa na mpesa"
+
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "BusinessShortCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "BusinessShortCode"
+                },
+                "Password": {
+                  "type": "string",
+                  "description": "Password"
+                },
+                "Timestamp": {
+                  "type": "string",
+                  "description": "Timestamp"
+                },
+                "TransactionType": {
+                  "type": "string",
+                  "description": "TransactionType"
+                },
+                "Amount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Amount"
+                },
+                "PartyA": {
+                  "type": "integer",
+                  "format": "int64",
+                  "description": "PartyA"
+                },
+                "PartyB": {
+                  "type": "integer",
+                  "format": "int64",
+                  "description": "PartyB"
+                },
+                "PhoneNumber": {
+                  "type": "integer",
+                  "format": "int64",
+                  "description": "PhoneNumber",
+                  "title": ""
+                },
+                "CallBackURL": {
+                  "type": "string",
+                  "description": "CallBackURL"
+                },
+                "AccountReference": {
+                  "type": "string",
+                  "description": "AccountReference"
+                },
+                "TransactionDesc": {
+                  "type": "string",
+                  "description": "TransactionDesc"
+                }
+              },
+              "required": [
+                "PhoneNumber"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/mpesa/stkpushquery/v1/query": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```

Comment:
@johnmuchiriM empty schemas are not allowed. please define a proper schema. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses). 
Please fix all empty schemas in the swagger file.